content/about/notes/1.1/index.md

```diff
@@ -1,32 +1,101 @@
+- **Simpler Service Entry Resources**. Service entry resources are greatly simplified by reducing the need to bind virtual services to them.
+
+- TBD: note about sidecar and export_to API.
 
 ## Security
 
-- Deprecated `RbacConfig` replacing it with `ClusterRbacConfig` to implement the correct cluster scope.
-  Refer to our guide on [Migrating the `RbacConfig` to `ClusterRbacConfig`](/docs/setup/kubernetes/upgrade/#migrating-the-rbacconfig-to-clusterrbacconfig)
-  for migration instructions.
+- **Readines and Liveness Probes**. Istio now supports Kubernetes' HTTP readiness and liveness probes when mutual TLS is enabled.
+TBD: LINK TO DOC
+
+- **`ClusterRbacConfig`**. The `RbacConfig` resource is being replaced with the `ClusterRbacConfig` resource to implement the correct cluster scope.
+Refer to our guide on [Migrating the `RbacConfig` to `ClusterRbacConfig`](/docs/setup/kubernetes/upgrading-istio#migrating-the-rbacconfig-to-clusterrbacconfig)
+for migration instructions.
+
+## Multiclusters
+
+- **Non-Routable L3 Networks**. A single Istio control plane can now be used in multicluster environments with non-routable
+L3 networks.
+TBD: LINK
+
+- **Multiple Control Planes**. Multiple Istio control plane multicluster is now supported.
+TBD: LINK
+
+## Policies and telemetry
+
+- **Policy Checks Off By Default**. Policy checks are now turned off by default which improves performance for most customer scenarios.
+TBD: LINK
+
+- **Kiali**. [Kiali](https://www.kiali.io) replaces the [Service Graph addon](https://github.com/istio/istio/issues/9066) and provides
+a richer visualization experience. See the [Kiali Task](/docs/tasks/telemetry/kiali/) for more details.
+
+- **Reduced Overhead**. There have been several performance and scale improvements including:
+
+    - Significant reduction in default collection of Envoy-generated statistics (configurable via annotations)
+
+    - Added load-shedding functionality to Mixer workloads.
+
+    - Improved the protocol between Envoy and Mixer.
+
+- **Control Headers and Routing**. It's now possible to create adapters that can be used to influence
+an incoming request's headers and routing. TBD: LINK
+
+- **Out of Process Adapters**. The out-of-process adapter functionality is now ready for production
+use. TBD: LINK
+ 
+- **Tracing Improvements**. There have been many improvements in our overall tracing story:
+
+    - We now use 128 bit trace ids
+    - There are some enhanced deployment options for Jaeger and Zipkin TBD: LINKS
+    - We now support sending trace data to LightStep
+    - Enabled full disablement of tracing for Mixer-backed services TBD: REPHRASE
+    - Added policy decision-aware tracing. TBD: LINK
+
+- **Default TCP Metrics**. There are now default metrics for tracking TCP connections.
```

Comment:
should we expand this to include work to add labels like `response_flags` to the default metrics (which enables better monitoring of Fault Injections and Circuit Breaking) ?

Reply:
Yes. Please recommend the right text.